### PR TITLE
fix: Update web notification from gtmpl to vue - EXO-66413

### PR DESCRIPTION
**Pull request by hakermi**
This PR updates the web notification old gtmpl templates to vue & vuetify for better UX

### CODE REVIEW

File: processes-services/src/main/java/org/exoplatform/processes/listener/CancelRequestNotificationListener.java [hunk 1]
```diff
@@ -18,6 +18,8 @@ public void onEvent(Event<TaskDto, ProjectDto> event) {
     ProjectDto projectDto = event.getData();
     NotificationContext ctx = NotificationContextImpl.cloneInstance();
     ctx.append(NotificationArguments.REQUEST_CREATOR, taskDto.getCreatedBy());
+    ctx.append(NotificationArguments.REQUEST_PROCESS, projectDto.getName());
+    ctx.append(NotificationArguments.REQUEST_TITLE, taskDto.getTitle());
```
Comment (rdenarie) [inline processes-services/src/main/java/org/exoplatform/processes/listener/CancelRequestNotificationListener.java:22]: I see you add parameters in the context of the notification.

So, notifications created BEFORE your modification will not have the same parameters than notification created AFTER your modifications

Can you ensure that notifications created BEFORE will not break the interface (js error for example) at least. In the better case, notifications created before can display a small set of informations even if all parameters are not present

Comment (hakermi) [reply processes-services/src/main/java/org/exoplatform/processes/listener/CancelRequestNotificationListener.java:22]: No js error or display break, i added new params only for CancelRequestPlugin but here is the display 

old notif display =>
cancelnotifbefore

new notif display
cancel_notif_after

Comment (hakermi) [reply processes-services/src/main/java/org/exoplatform/processes/listener/CancelRequestNotificationListener.java:22]: we can improve it by adding condition on existing params and display the old message. wdyt?

Comment (rdenarie) [reply processes-services/src/main/java/org/exoplatform/processes/listener/CancelRequestNotificationListener.java:22]: yes please, if it is possible I prefer to make both possibility works

Comment (hakermi) [reply processes-services/src/main/java/org/exoplatform/processes/listener/CancelRequestNotificationListener.java:22]: done https://github.com/exoplatform/processes/compare/30fc51145f9633622e02e9791ae48e7f85c0b3ac..4d7f083a359ac61e82f4c7bcf557a3e5cfe91f85

Comment (Jihed525) [reply processes-services/src/main/java/org/exoplatform/processes/listener/CancelRequestNotificationListener.java:22]: @rdenarie is  ok on your part following @hakermi latest feedback ?

Comment (rdenarie) [reply processes-services/src/main/java/org/exoplatform/processes/listener/CancelRequestNotificationListener.java:22]: yes it is

**Review by rdenarie — DISMISSED** [review]

**Review by Jihed525 — APPROVED** [review]

**Pull request merged**